formidable/views.py

```diff
@@ -290,3 +293,32 @@ def get_form(self, form_class):
 
     def get_form_kwargs(self):
         return {'data': self.request.GET}
+
+
+class ValidateViewFromSchema(ValidateView):
+    """
+    Acts like `ValidateView` but it uses a Formidable JSON schema instead of a
+    `Formidable` object.
+
+    """
+
+    settings_permission_key = 'FORMIDABLE_PERMISSION_USING'
+
+    def get_formidable_object(self):
+        raise NotImplemented
+
+    class ValidationFileFieldBuilder(field_builder_from_schema.FileFieldBuilder):  # noqa
+
+        def build(self, *args, **kwargs):
+            raise SkipField
+
+    def get_form_class(self, formidable):
+
```

Comment:
docstrings here (or maybe comments) to explain how this ``get_form_class`` differs from the ValidateView one.